web-ui/src/main/resources/catalog/views/default/templates/searchForm.html

```diff
@@ -34,17 +35,22 @@
 
                 <button type="button"
                         data-ng-click="triggerSearch()"
+                        title="{{'search' | translate}}"
                         class="btn btn-primary btn-lg">
                   &nbsp;&nbsp;
                   <i class="fa fa-search"></i>
                   <span class="sr-only" data-translate="">search</span>
                   &nbsp;&nbsp;
                 </button>
+
+                <button data-gn-featured-user-searches-home=""
```

Comment:
Maybe can be updated the name of the directive to `gFeaturedUserSearches`.

Reply:
Done gnUserSearchesList as it may be used to list featured or other type of user search.